### DISCUSSION TITLE
Front-end user application notifications

### DISCUSSION THREAD

**Discussion by kellyannmcnamara01** [craftcms/cms#14375] [Ideas]
**The Current**
We have a website that allows membership applications through the front-end. These applications create disabled users in the admin and the client goes through and reviews the application prior to having the member appear in their site

**The Problem**
The client has to manually go into the admin panel and look at the user group to see if new applicants have applied. We have unapproved members in a different user group but from my understanding the client doesn't log into their admin daily

**The Request**
Allow a setting in /admin/settings/users/settings to add an email address for the client to be notified every time a new membership application has been submitted

**Comment by leevigraham** [comment]
You could add this as an after save event: https://craftcms.com/docs/4.x/extend/element-types.html#save-hooks
